### PR TITLE
add table to define photo naming

### PR DESCRIPTION
**Pull request by 3nids**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -164,5 +164,6 @@ fabric.properties
 # My super new category
 .DS_Store
 
-resources_rc.py
+qfield-sync.*.zip
```
Comment (m-kuhn) [inline .gitignore]: Is the `-` (dash) on purpose?

File: qfieldsync/utils/exceptions.py [hunk 2]
```diff
@@ -19,17 +19,12 @@
  ***************************************************************************/
 """
 
-# coding=utf-8
-from qgis.core import (
-        QgsMessageLog,
-        Qgis
-)
+from qgis.core import QgsMessageLog, Qgis
```
Comment (m-kuhn) [inline qfieldsync/utils/exceptions.py]: The idea of the multiline import is to keep them trackable in git

File: .gitignore (hunk 1 above)
Comment (3nids) [reply .gitignore]: yep

File: qfieldsync/utils/exceptions.py (hunk 2 above)
Comment (3nids) [reply qfieldsync/utils/exceptions.py]: the IDE makes it clear which is used or not.
is there another purpose?

Comment (m-kuhn) [reply qfieldsync/utils/exceptions.py]: It's about git history, not the ide

Comment (3nids) [reply qfieldsync/utils/exceptions.py]: I just don't see the interest in tracking import changes and I prefer the cleaner code layout.
But I don't mind reverting, just tell me

Comment (m-kuhn) [reply qfieldsync/utils/exceptions.py]: I'd like to have a common style here, I was always switching things TO this style, but switching forth and back seems counterproductive.

We could also go for

```
from x import Y
from x import Z
```

that would also avoid extremely long lines

**Comment by signedav** [comment]
I had to create the resources.py manually.  
`pyrcc5 resources.qrc -o resources_rc.py`
Is there missing something or is it just me?

**Comment by m-kuhn** [comment]
You could have used `make`. In the old days we used to "compile" images and dialogs into resource py files, as we also do with cpp projects still often. This defeats one of the nice things of python (no compilation involved) so we no longer do this a lot nowadays.

**Pull request merged**